src/operations/find_and_modify.ts

```diff
@@ -29,6 +29,8 @@ export interface FindOneAndDeleteOptions extends CommandOperationOptions {
   sort?: Sort;
   /** Map of parameter names and values that can be accessed using $$var (requires MongoDB 5.0). */
   let?: Document;
+  /** Return the raw result document instead of the ModifyResult */
+  returnRawResult?: boolean;
```

Comment:
Just wanted to bring a couple of points to your attention from the ticket description so that we minimize potential back and forth in review:

> Dev note: `returnRawResult` is not meant to be prescriptive - the server returns a write_ops::FindAndModifyCommandReply, internally the driver returns a ModifyResult but our flag should make it clearer as to what is being returned

> API docs should clearly explain the option

It may be worth taking this back to the team for consensus on option name; once the option name is settled, it's important to make sure we set accurate expectations for what the option does. The spec compliant version of these commands returns less information than the current `ModifyResult`. In the original ticket, `returnRawResult = true` is intended to return what is currently `ModifyResult` and `returnRawResult = false` is intended to return the spec compliant version, which is equal to `ModifyResult.value`.